File: golangtodo/src/todo.go
package src

import (
	"time"
)

type Todo struct {
	ID           uint      `gorm:"primary_key" json:"id"`
	Task         string    `json:"task"`
	Description  string    `json:"description"`
	Completed    bool      `json:"completed"`
	Edited       bool      `json:"edited"`
	SerialNumber int       `json:"serialNumber"`
	LastViewed   bool      `json:"lastViewed"`
	CreatedDate  time.Time `json:"createdDate"`
	ModifiedDate time.Time `json:"modifiedDate"`
}

File: phptodo/src/todo.php
<?php
class Todo
{
  public $id;
  public $task;
  public $description;
  public $completed;
  public $edited;
  public $serialNumber;
  public $lastViewed;
  public $createdDate;
  public $modifiedDate;

  public function __construct($id, $task, $description, $completed, $edited, $serialNumber, $lastViewed, $createdDate, $modifiedDate)
  {
    $this->id = $id;
    $this->task = $task;
    $this->description = $description;
    $this->completed = $completed;
    $this->edited = $edited;
    $this->serialNumber = $serialNumber;
    $this->lastViewed = $lastViewed;
    $this->createdDate = $createdDate;
    $this->modifiedDate = $modifiedDate;
  }
}
